.travis.yml

```diff
@@ -66,10 +66,17 @@ matrix:
       - IMG=rocm
       - CMAKE_EXTRA_FLAGS="-C ../host-configs/rocm.cmake -DCMAKE_CXX_COMPILER=/opt/rocm/bin/hcc -DENABLE_HCC=On -DENABLE_OPENMP=Off -DENABLE_CUDA=Off -DENABLE_BENCHMARKS=Off -DENABLE_WARNINGS_AS_ERRORS=Off"
       - DO_TEST=no
+  - compiler: hip
+    env:
+      - COMPILER=g++
+      - IMG=rocm
+      - HCC_AMDGPU_TARGET=gfx900
+      - CMAKE_EXTRA_FLAGS="-DENABLE_HIP=On -DENABLE_OPENMP=Off -DENABLE_CUDA=Off -DENABLE_WARNINGS_AS_ERRORS=Off"
+      - DO_TEST=no
```

Comment:
What CI are we getting with testing disabled?

Reply:
Never mind.  I see that it is just the HIP build that is being done (from the CHANGELOG).